### PR TITLE
Automated sending

### PR DESCRIPTION
**Pull request by jsf9k**
This pull request contains the code to automate the emailing of Trustworthy Email, HTTPS, and CyHy reports as well as Cyber Exposure scorecards.  This feature works when the code is run via `docker-compose` from NCATS' production and staging servers.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -7,6 +7,21 @@ secrets:
...
+        target: /cyhy_reports
+      - type: bind
+        source: /var/cyhy/orchestrator/output/archive/latest/reporting/trustymail_reports
+        target: /trustymail_reports
+      - type: bind
+        source: /var/cyhy/orchestrator/output/archive/latest/reporting/pshtt_reports
+        target: /pshtt_reports
+      - type: bind
+        source: /var/cyhy/weekly_reporting/scorecards/latest
+        target: /cybex_report
```
Comment (dav3r) [inline docker-compose.yml]: "cybex_report" should probably be "cybex_scorecard", in order to reinforce the semantic difference between reports (focused on a single stakeholder) and scorecards (summarize data across multiple stakeholders).

File: docker-compose.yml [hunk 2]
```diff
@@ -7,6 +7,21 @@ secrets:
...
+    command: ["report", "--cyhy-report-dir=/cyhy_reports/", "--cybex-report-dir=/cybex_report/", "--tmail-report-dir=/trustymail_reports/", "--https-report-dir=/pshtt_reports/", "--db-creds-file=/run/secrets/database_creds.yml", "--summary-to=ncats@hq.dhs.gov,jeremy.frasier@beta.dhs.gov"]
```
Comment (dav3r) [inline docker-compose.yml]: "--cybex-report-dir=/cybex_report/" -->  "--cybex-scorecard-dir=/cybex_scorecard/" (see comment above)

**Review by dav3r — CHANGES_REQUESTED** [review]
Looks good, except for a minor language gripe

File: docker-compose.yml (hunk 1 above)
Comment (jsf9k) [reply docker-compose.yml]: OK, I can change that.

**Comment by jsf9k** [comment]
Have a look, @dav3r.  It should be good now.

**Review by dav3r — APPROVED** [review]

**Pull request merged**